src/Tribe/Views/V2/Url.php

```diff
@@ -285,6 +285,16 @@ public static function from_url_and_params( $url = null, array $params = [] ) {
 			$url = home_url( add_query_arg( [] ) );
 		}
 
+		/*
+		 * Depending on the source of the request the encoded chars might be in uppercase (`%C3`) or lowercase (`%c2j`) format.
+		 * Ensure the format of the encoded chars is consistently lowercase.
+		 */
+		$lowercase_encoded_url = preg_replace_callback( '/%[A-F0-9]{2}/', static function ( array $matches ) {
+			return strtolower( reset( $matches ) );
+		}, $url );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 8e6e5e73e0eeb82f8eba36ba0a7d54ad</sub>